packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -336,6 +336,18 @@ export const getVideoReceiverStreamMqa = ({
     statsResults[mediaType][sendrecvType].keyFramesDecoded - lastKeyFramesDecoded || 0;
   videoReceiverStream.requestedKeyFrames =
     statsResults[mediaType][sendrecvType].totalPliCount - lastPliCount || 0;
+
+  if (statsResults[mediaType][sendrecvType].isActiveSpeaker !== undefined) {
+    if (statsResults[mediaType][sendrecvType].lastActiveSpeakerTimestamp !== undefined) {
+      const currentTimestamp =
+        statsResults[mediaType][sendrecvType].lastActiveSpeakerTimestamp + MQA_INTERVAL;
+      // check if last true active speaker was set at least once during the last MQA interval
+      videoReceiverStream.isActiveSpeaker =
+        currentTimestamp > performance.now() + performance.timeOrigin;
```

Comment:
This calculation is confusing to me, probably because `currentTimestamp` sounds like it should be the time that it is right now.

So I would personally write it this way:

```
const currentTimestamp = performance.now() + performance.timeOrigin;
videoReceiverStream.isActiveSpeaker = statsResults[mediaType][sendrecvType].lastActiveSpeakerTimestamp > currentTimestamp - MQA_INTERVAL;
```

Let's also write some unit tests for this to make sure we cover all cases, since this is a bit tricky. In particular:
- A user was constantly an active speaker for the last 5 minutes and did not change their active speaker state within the last interval (`isActiveSpeaker` MQE should be true)
- A user was never an active speaker for the last 5 minutes and did not change their active speaker state within the last interval (`isActiveSpeaker` MQE should be false)
- A user changed their active speaker state within the last interval (`isActiveSpeaker` MQE should be true)

Reply:
@brycetham Thank you for the ideas. I agree with you on the naming. Your suggestion also seems more natural and logical, so I will make the change. Regarding tests, I plan to add them after your review, so your test cases will also be useful to consider. This logic definitely should be covered with tests. Thanks for the review.